sdk/core/azure-core/src/environment_log_level_listener.cpp

```diff
@@ -144,4 +144,9 @@ EnvironmentLogLevelListener::GetLogListener()
   return consoleLogger;
 }
 
+static bool initialized;
```

Comment:
Please move it to unnamed namespace and name it `g_initialized` / `g_isInitialized`

Reply:
Does it need the DLL EXPORT?

Reply:
nope - DLLEXPORT is for static data members in .hpp files.

Reply:
done